apps/eventsub/internal/manager/manager.go

```diff
@@ -58,7 +57,58 @@ func NewManager(opts ManagerOpts) (*Manager, error) {
 	opts.Lc.Append(
 		fx.Hook{
 			OnStart: func(ctx context.Context) error {
+				if opts.Config.AppEnv != "production" {
+					if err := manager.
+						gorm.
+						Session(&gorm.Session{AllowGlobalUpdate: true}).
+						Delete(&model.EventsubSubscription{}).
+						Error; err != nil {
+						return err
+					}
+				}
+
 				go func() {
+					if opts.Config.AppEnv != "production" {
+						twitchClient, err := twitch.NewAppClient(opts.Config, opts.TokensGrpc)
+						if err != nil {
+							panic(err)
+						}
+
+						var subscriptions []helix.EventSubSubscription
+						cursor := ""
+						for {
+							subs, err := twitchClient.GetEventSubSubscriptions(
+								&helix.EventSubSubscriptionsParams{
+									After: cursor,
+								},
+							)
+							if err != nil {
+								panic(err)
+							}
+
+							subscriptions = append(subscriptions, subs.Data.EventSubSubscriptions...)
+
+							if subs.Data.Pagination.Cursor == "" {
+								break
+							}
+
+							cursor = subs.Data.Pagination.Cursor
+						}
+
+						var unsubWg sync.WaitGroup
+
+						for _, sub := range subscriptions {
+							sub := sub
```

Comment:
I think it's better to move service to workspace go version 1.22.2 and remove redundant line.